Resources/Prototypes/_Floof/Entities/Mobs/NPCs/clown_abominations.yml

```
@@ -0,0 +1,324 @@
+      - Syndicate
+  - type: HTN
+    rootTask:
+      task: SimpleHostileCompound
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Slash
+        damage: 60
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Blunt
+        damage: 60
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Piercing
+        damage: 60
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+  
+- type: entity
+  name: crazed clown rat
+  parent: [ MobClownRatHatchling , MobCombat ]
+  id: MobCrazedClownRatHatchling
+  description: OH GOD THEY ARE MULTIPLYING!
+  suffix: Syndicate, Hatchling
+  components:
+  - type: NpcFactionMember
+    factions:
+      - Syndicate
+  - type: HTN
+    rootTask:
+      task: SimpleHostileCompound
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Slash
+        damage: 40
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatFinalHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Blunt
+        damage: 40
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatFinalHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Piercing
+        damage: 40
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawnInContainer: true
+        spawn:
+          MobCrazedClownRatFinalHatchling:
+            min: 3
+            max: 4
+      - !type:GibBehavior { }
+  
+- type: entity
+  name: crazed clown rat
+  parent: [ MobClownRatFinalHatchling, MobCombat ]
+  id: MobCrazedClownRatFinalHatchling
+  description: GOD THERE IS EVEN MORE!
+  suffix: Syndicate, Final_Hatchling
+  components:
+  - type: NpcFactionMember
+    factions:
+      - Syndicate
+  - type: HTN
+    rootTask:
+      task: SimpleHostileCompound
+      
+- type: entity
+  name: dehydrated crazed clown rat box
+  parent: DehydratedSpaceCarp
+  id: DehydratedClownRatBox
+  description: The malice of the honk mother weponised by the syndicate in a compact package. Just pet its noggin and add water!
```

```suggestion
  description: The malice of the honkmother, weaponized by the syndicate in a compact package. Just pet its noggin and add water!
```
After a quick lookup, it turns out whether or not "honkmother" is a proper noun or common noun is... inconsistent. But it is always spelled as "honkmother" instead of "honk mother". 

This can be observed as "Honkmother" in  [`instruments_misc.yml` at line 156](https://github.com/Fansana/floofstation1/blob/fc592af2975b7db64772e966c127e28388d3b002/Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_misc.yml#L156), and "honkmother" in [`ghost-role-component.ftl` at line 218](https://github.com/Fansana/floofstation1/blob/fc592af2975b7db64772e966c127e28388d3b002/Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl#L218).

---

Good to know thak you for spell checking it i ussly do it as the last thing after all the bugs are ironed out, so i do appriate it. will commit the change